packages/cojson-storage-indexeddb/src/idbClient.ts

```diff
@@ -10,30 +10,25 @@ import type {
 } from "cojson-storage";
 import { SyncPromise } from "./syncPromises.js";
 
+type TxEntry = {
+  id: number;
+  tx: IDBTransaction;
+  stores: {
+    coValues: IDBObjectStore;
+    sessions: IDBObjectStore;
+    transactions: IDBObjectStore;
+    signatureAfter: IDBObjectStore;
+  };
+  running: boolean;
+  startedAt: number;
+  isControlledTransaction?: boolean;
+  pendingRequests: ((txEntry: TxEntry) => void)[];
+};
+
 export class IDBClient implements DBClientInterface {
```

Comment:
Need to add a test to verify what happens if the same content message is sent twice